website/content/docs/bloop-basics.md

```diff
@@ -0,0 +1,127 @@
++++
+toc = true
+weight = 2
+draft = false
+title = "The basics of bloop"
+date = "2018-02-09T10:15:00+01:00"
+description = "What is bloop and which problems does it solve?"
+bref = "Learn the ideas behind bloop, which problems solve and how it's designed."
++++
+
+## An overview of Bloop
+
+Bloop is a Scala command-line tool and build server developed at the Scala
+Center to make the compile and test feedback loop as tight as possible. This
+means a faster and more productive developer workflow.
+
+### A common pain point
+
+Edit, compile and test workflows are the bread and butter of our daily jobs.
+When our build is slow to respond, our productivity drops. Can Scala get as
+tight edit/compile/test workflows as in Clojure or OCaml? Bloop is a
+command-line tool and build server that brings you a tighter developer
+workflow. Slow Scala compile times can often be attributed to the slugishness
+of our build tool, and `bloop` finds a solution to them.
+
+We have created bloop to make you productive without getting in your way. We
+focus on how you can be faster at writing Scala code using your current build
+tool, whether it’s sbt, Maven or Gradle—without reinventing the wheel with a
+new build tool.
+
+### The main goal
+
+The primary goal of bloop is to keep your hot compilers around.
```

Comment:
The primary goal is to have better performance, keeping hot compilers is a mean to achieve that goal. Nobody cares about having a hot or cold compiler, they just want it to be fast.

Reply:
> The primary goal is to have better performance, keeping hot compilers is a mean to achieve that goal.
  
That is described above. Keeping hot compilers is the most important thing a tool can do to be fast at compiling code. We cannot claim to have better performance by just saying we "do less" than other build tools.
  
Keeping hot compilers across all editors and build tools is the key aspect that distinguishes bloop from, say, sbt or mill.

Reply:
@Duhemm I'm withdrawing my previous claim, I think you're right -- no matter what the details to be fast are, our primary focus is to be faster than the rest.